src/components/elementView/detailedView/decommissionedPropertiesView.tsx

```diff
@@ -0,0 +1,74 @@
+import { Classes } from "@blueprintjs/core";
+import "@blueprintjs/core/lib/css/blueprint.css";
+import * as React from "react";
+import { RouteComponentProps, withRouter } from "react-router";
+import {
+    ElementObjectType,
+    DecommissionedFieldsTable,
+} from "../../../utils/utils";
+
+interface DecommissionedPropertiesViewProps {
+    data: ElementObjectType;
```

Comment:
Can you ever decommission anything other than an asset? If not, it's probably safe to just have the type of this as AssetObject or whatever we use to reference assets on the front end

Reply:
Good looks I'll change

Reply:
Done!